hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2434,13 +2434,18 @@
     {
       name: chip_sw_keymgr_sideload_kmac_error
       desc: '''
-            Verify the effect of KMAC returning an error during keymgr sideload operation.
+            Verify the effect of KMAC returning an error during a keymgr operation.
 
-            - Follow the steps in `chip_sw_keymgr_sideload_kmac` test.
+            - Configure keymgr to enter any of the 3 working states.
+            - Issue a keymgr operations.
             - While the KMAC is actively computing the digest, glitch the KMAC app sparse FSM to
               trigger a fault.
-            - Verify that KMAC returns an error signal to the keymgr.
-            - TBD
+            - Verify that KMAC returns an error signal to the keymgr via checking keymgr CSRs, when
+              the operation is done:
+              - Check `op_status` is set to `DONE_ERROR`.
+              - Check `fault_status.kmac_done` is set to 1.
+            - Note: at end of the C test, issue a reset if the test is running in DV, otherwise, the
+              fatal alert will prevent the simulation from finishing.
```

Comment:
Just to clarify if I understand it correctly:

1. Error at KMAC and fault at Keymgr does *not* create an alert that blocks Ibex,
2. Therefore, we can perform reset from SW side.

Reply:
hmmm... performing a reset just for that here seems a bit overkill though.
@weicaiyang couldn't we use the plusarg to just allow hanging alerts?

Reply:
Both Kmac and keymgr will fire alerts, but the alerts shouldn't block ibex.

we could use this plusarg too. Let me remove this note. either a reset or plusarg is fine to me.
```
run_opts: ["+bypass_alert_ready_to_end_check=1"]
```